CONTRIBUTING.md

```diff
@@ -22,7 +22,7 @@ Please make sure you go through this list before submitting a patch.  The rules
 
 - [ ] Did you check your code is formatted correctly? It's easiest to run
 ````
-find src include -name '*.cc' -or -name '*.h' -or -name '*.c'|xargs clang-format -i -style=file
+find src include -name '*.cc' -or -name '*.h' -or -name '*.c'|fgrep -v proc_service.h|xargs clang-format -i -style=file
```

Comment:
... in its own subdirectory, with a README explaining where you copied it and what must be done to comply with its license.